services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -4985,7 +4985,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/put/responses/200/content/application~1json/schema'
+                $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/get/responses/200/content/application~1json/schema'
```

Comment:
curious that the generator takes now the ``get`` instead of the ``put`` path to find the schema. I guess this has to do with the version issue we saw some days ago.

Reply:
yes I'll regenerate it again at the office to be sure...